datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -496,6 +527,175 @@ internal class QuestionnaireViewModel(application: Application, state: SavedStat
 
     return null
   }
+
+  /**
+   * This function evaluates variable expression and return the evaluated result. To evaluate root
```

Comment:
Typically we just use third-person singular form to write documentation for functions.

```suggestion
   * Evaluates variable expression and return the evaluated result. To evaluate root
```

Same with all other functions in this pr

Reply:
Done.